static/app/views/alerts/rules/metric/details/body.tsx

```diff
@@ -66,6 +70,10 @@ export default function MetricDetailsBody({
   const organization = useOrganization();
   const location = useLocation();
   const navigate = useNavigate();
+  const [showTransactionsDeprecationAlert, setShowTransactionsDeprecationAlert] =
+    useState(
+      organization.features.includes('performance-transaction-deprecation-banner')
+    );
```

Comment:
Often times we make these permanently dismissible (ie it persists or refresh). It looks like that won't be the case here and want to confirm we're okay with that.

Reply:
I'll follow up but I think we're okay with these being temporarily dismissed because we want to make sure it's not something users are forgetting to try out

Reply:
Confirming, we're okay with not using permanently dismissible warnings

Reply:
Perfect!